src/main/java/mate/academy/spring/model/dto/MovieSessionRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.spring.model.dto;
+
+public class MovieSessionRequestDto {
+    private Long id;
+
```

Comment:
Привіт, ти трохи перевиконав те, що просив Олег. Він мав на увазі, що ми не маємо виставляти наші моделі назовні щоб мати більш захищене рішення. Але для логіки нам загалом потрібне розуміння з яким фільмом і залом пов'язаний запит, тому треба було замінити моделі на відповідні id, а не повністю їх випилювати.
Якщо чесно, то я зараз навіть не дуже розумію як може коректно працювати застосунок без отримання цієї інформації
```suggestion
public class MovieSessionRequestDto {
    private Long movieId;
    private Long cinemaHallId;
    private LocalDateTime showTime;

```